### ISSUE TITLE
Infinite loop trying to plug RapidFuzz into a Flask API

### ISSUE THREAD

**Issue by fbexiga** [rapidfuzz/RapidFuzz#95]
I'm trying to create an endpoint that does some work based on RapidFuzz (which is beyond awesome). But as soon as I import the package, it enters an infinite loop.

```
from flask import Flask, Response
from rapidfuzz import process

app = application = Flask(__name__)

class HelloWorld:
    @classmethod
    def on_get(cls):
        return Response('\r\n', mimetype='application/json')

app.add_url_rule('/', '/', view_func=HelloWorld.on_get)
```

and running it as 
```gunicorn --bind 0.0.0.0:8080 --worker-class gevent --workers 4 --threads 2 testapp```

Any way I can make this work?

**Comment by maxbachmann** [comment]
I could reproduce the issue. The bug was introduces in Version 1.3.1. In this release I started to use cython 3.0a6 instead of 0.29.21 to generate my C++ files. The bug does not occur when generating the files using version 0.29.21, so this is a bug in Cython.
As a temporary fix you can use v1.3.0.

**Comment by maxbachmann** [comment]
Do you happen to know, whether there is a way to dump the stacktrace of a crashing gunicorn worker?

**Comment by fbexiga** [comment]
Thanks! I was hoping to be able to use some of the latest features but this is great already!

I can't say I know a way do dump gunicorn stacktrace but I can try looking into it. At this point I'm not even sure if gunicorn or gevent itself is where I should be looking at.

**Comment by da-woods** [comment]
An easy way of getting some extra information is just to use the faulthander module (https://docs.python.org/3/library/faulthandler.html). It may not give you the full C traceback but it's definitely a start.

**Comment by maxbachmann** [comment]
@fbexiga I fixed the issue in v1.4.1 https://github.com/maxbachmann/RapidFuzz/releases/tag/v1.4.1
